src/assets/GraphqlAsset.js

```diff
@@ -1,15 +1,62 @@
 const Asset = require('../Asset');
 const localRequire = require('../utils/localRequire');
+const Resolver = require('../Resolver');
+const fs = require('../utils/fs');
+const os = require('os');
+
+const IMPORT_RE = /^# *import +['"](.*)['"] *;? *$/;
 
 class GraphqlAsset extends Asset {
   constructor(name, options) {
     super(name, options);
     this.type = 'js';
+
+    this.gqlMap = new Map();
+    this.gqlResolver = new Resolver(
+      Object.assign({}, this.options, {
+        extensions: ['.gql', '.graphql']
+      })
+    );
+  }
+
+  async traverseImports(name, code) {
+    this.gqlMap.set(name, code);
+
+    await Promise.all(
+      code
+        .split(os.EOL)
```

Comment:
Will this work in every case?
Seems like this would not work if a user clones a repo that is being used developed various OSes

Reply:
Good point, using a regex such as `/\r?\n/` is probably resilient.

Reply:
Changed it to use `/\r\n?|\n` which is pretty much the same as [what the GraphQL specification defines as a line terminator](http://facebook.github.io/graphql/June2018/#sec-Line-Terminators)